cmd/medius/images/verify.go

```diff
@@ -55,7 +55,9 @@ func NewVerifyImagesCommand(options *common.Options) *cobra.Command {
 			focusMatched, resultsChan, workerErr := spawnWorkers(cmd.Context(), options, func(e *common.Entry) (*api.ArtifactResult, error) {
 				artifact, err := retrieveArchitectureArtifact(options, e)
 				if err != nil {
-					return nil, err
+					firstArtifactMedatada := e.Artifacts[0].Metadata()
+					logrus.Warn(firstArtifactMedatada.Name + ":" + firstArtifactMedatada.Version + " - " + err.Error())
```

Comment:
Should that state it is skipping the image but continuing?

Reply:
Exactly, in the case of Fedora images, where some are compatible with s390x and others are not, only the valid images undergo verification and testing, while the incompatible ones are skipped.

Reply:
I meant, can you update the log message to say so? :)